joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/undertow/UndertowAutoConfiguration.java

```diff
@@ -57,8 +57,9 @@
 				ClassLoader jsfClassLoader = new URLClassLoader(new URL[0], this.getClass().getClassLoader());
```

Comment:
Instead of creating our own classloader here, we could use the one selected by spring boot by calling `deploymentInfo.getClassLoader()`

Reply:
Ok. I will test it.

Reply:
done.